test/db/analysis/avr

```diff
@@ -673,36 +672,36 @@ EXPECT=<<EOF
 0x00000158 13478 entry0
 0x00000158 2 pcl
 0x000001ba 188 fcn.000001ba
-0x000001d0 1 syscall.read.2
-0x00000260 1 syscall.read.3
+0x000001d0 1 syscall.0.2
+0x00000260 1 syscall.0.3
 0x0000029e 164 fcn.0000029e
-0x000002ae 1 syscall.read.5
-0x000002f2 1 syscall.inotify_rm_watch.1
-0x00000318 1 syscall.inotify_rm_watch.0
-0x00000330 1 syscall.inotify_rm_watch
+0x000002ae 1 syscall.0.5
```

Comment:
I'm not sure I understand why these need to be changed. Can you explain?

Reply:
There are no such syscalls on the AVR architecture.

Reply:
There are, I guess, if you have specified os=linux as it was before.
Was this test just completely wrong?

Also, are there syscalls anyway in those places?

Reply:
yes, the test was wrong.